### PR TITLE
Document case management workflows

### PR DESCRIPTION
**Pull request by garethbowen**
medic/cht-core#6291

### CODE REVIEW

**Comment by garethbowen** [comment]
Somehow this medic-docs commit got missed: https://github.com/medic/medic-docs/commit/436471252146d946493afb8ad2b0616c51c3f99c

**Comment by garethbowen** [comment]
I've made the documentation say "accept_case_reports" everywhere.

File: content/en/apps/reference/app-settings/transitions.md [hunk 1]
```diff
@@ -436,6 +441,32 @@ Supported `events_types` are:
...
+- validating the report configuration documentation,
+- assigning the relevant place to the report if a registration with the given case_id exists,
+- clearing messages on the registration, and
+- generating response messages on the report.
+
+#### Example
+
+```json
+"registrations": [{
+  "form": "8"
```
Comment (abbyad) [inline content/en/apps/reference/app-settings/transitions.md:456]: If the registration is included in the example, should it have `add_case`?

Comment (garethbowen) [reply content/en/apps/reference/app-settings/transitions.md]: Yes, this was a truncated example. I've added the event config as well to avoid confusion.

**Pull request merged**